src/test/resources/perf/binarytrees.out

```diff
@@ -1,4 +1,8 @@
-stretch tree of depth 7	 check: -1
```

Comment:
I'm so confused why the output of this has changed. Is there an obvious change I've missed?

Reply:
`PerformanceTest` and `PerformanceBenchmark` never actually verify the correctness of the programs' output. When I added `helpers.runComparisonTest(name)` to `PerformanceTest` on `master`, I was surprised to see that the test failed for every single benchmark -- the `.out` files did not match the actual output. Since I wanted behaviour identical to `master`, I replaced the test resources with the output I got from there.